.github/workflows/main.yml

```diff
@@ -94,11 +97,14 @@ jobs:
     uses: ./.github/workflows/support-packages.yml
     with:
       branch-name: ${{ needs.init.outputs.branch-name }}
-      drivers-bucket: ${{ needs.init.outputs.drivers-bucket }}
+      upstream-drivers-bucket: ${{ needs.init.outputs.drivers-bucket }}
+      downstream-drivers-bucket: ${{ needs.init.outputs.cpaas-drivers-bucket }}/x86_64
       support-packages-bucket: ${{ needs.init.outputs.support-packages-bucket }}
       support-packages-index-bucket: ${{ needs.init.outputs.support-packages-index-bucket }}
       public-support-packages-bucket: ${{ needs.init.outputs.public-support-packages-bucket }}
-    if: always() && (github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'test-support-packages'))
+    if: |
+      (success() || failure()) &&
```

Comment:
Perhaps this needs a comment. It looks like this might always evaluate to true.

Reply:
I'll add one, the reason for this change is that cancelling a workflow still ran the support-package step, which shouldn't be the case, this new condition skips the step when the workflow is cancelled but still runs it if everything is fine or a previous step failed.